transmart-rest-api/grails-app/services/org/transmartproject/rest/dataExport/ExportService.groovy

```diff
@@ -0,0 +1,170 @@
+package org.transmartproject.rest.dataExport
+
+import grails.transaction.Transactional
+import org.quartz.JobBuilder
+import org.quartz.JobDataMap
+import org.quartz.JobDetail
+import org.quartz.Scheduler
+import org.quartz.TriggerBuilder
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+import org.transmartproject.core.exceptions.InvalidRequestException
+import org.transmartproject.db.job.AsyncJobCoreDb as AsyncJob
+import org.transmartproject.db.user.User
+
+import javax.transaction.NotSupportedException
+
+@Transactional
+@Component("restExportService")
```

Comment:
yes, without this I was getting BeanCreationException: "Error creating bean with name 'org.transmartproject.rest.ExportController': Initialization of bean failed; nested exception is org.springframework.beans.ConversionNotSupportedException: Failed to convert property value of type [com.recomdata.transmart.data.export.ExportService] to required type [org.transmartproject.rest.dataExport.ExportService] for property 'exportService'; nested exception is java.lang.IllegalStateException: Cannot convert value of type [com.recomdata.transmart.data.export.ExportService] to required type [org.transmartproject.rest.dataExport.ExportService] for property 'exportService': no matching editors or conversion strategy found"